### PR TITLE
fix(hydra-show-monitor): add call to `hydra attach-test-sg`

### PR DESCRIPTION
**Pull request by fruch**
since we move to close security groups by default we need to create a security group, for the newly monitor node created in this pipeline

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sct_ssh.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def other(event):
     return question.ask()
 
 
-def select_instance_group(region: str = None, **tags) -> dict | None:
+def select_instance_group(region: str = None, **tags) -> list:
```
Comment (yarongilor) [inline sct_ssh.py:118]: is this related somehow to the show-monitor fix?

Comment (fruch) [reply sct_ssh.py:118]: yes,  without this fix it won't work.

i.e. there a bug here that if this list return only once instance, it didn't returned a list, and the command would iterate on it's keys, and not on the node details, and would fail...

**Comment by fruch** [comment]
testing it in:
https://jenkins.scylladb.com/job/QA-tools/job/hydra-show-monitor/327/
